gists_database/search.py

```diff
@@ -1,5 +1,52 @@
 from .models import Gist
+from datetime import datetime
+
+
+def operator(op):
+    d = {'gt': '>', 'gte': '>=', 'lt': '<', 'lte': '<='}
+    if op in d:
+        return d[op]
+    return '=='
 
 
 def search_gists(db_connection, **kwargs):
-    pass
+    global query
```

Comment:
Oh no! A global variable! You can definitively make this work without the global variable